eox_tenant/tenant_wise/proxies.py

```diff
@@ -176,12 +176,20 @@ def get_queryset(self):
         Call parent method and filter the certificates by org.
         """
         generated_certificates = super(TenantCertificateManager, self).get_queryset()
-        excluded_courses = [
-            certificate.course_id
-            for certificate in generated_certificates
-            if certificate.course_id.org not in getattr(settings, "course_org_filter", [])
-        ]
-        return generated_certificates.exclude(course_id__in=excluded_courses)
+        org_filter = getattr(settings, "course_org_filter", [])
```

Comment:
Should we store this code that gets the `orgs` from the current site in a model or a reusable module?
I believe we are going to use it quite often, but I'm not sure where would the best place be.